src/services/mentees.js

```diff
@@ -145,26 +173,62 @@ module.exports = class MenteesHelper {
 			communications,
 		}
 
-		if (!menteeDetails.data.result.organization) {
-			const orgDetails = await organisationExtensionQueries.findOne(
-				{ organization_code: organizationCode },
-				tenantCode,
-				{ attributes: ['name'] }
-			)
-			menteeDetails.data.result['organization'] = {
-				id: organizationCode,
-				name: orgDetails.name,
+		// Add missing computed fields to processDbResponse
+		processDbResponse.profile_mandatory_fields = profileMandatoryFields
+
+		// Add organization object
+		const orgDetails = await organisationExtensionQueries.findOne(
+			{ organization_code: organizationCode },
+			tenantCode,
+			{ attributes: ['name', 'organization_code', 'organization_id'] }
+		)
+		processDbResponse.organization = {
+			id: orgDetails.organization_code,
+			name: orgDetails.name,
+		}
+
+		// Add sessions_attended (both mentees and mentors can attend sessions)
+		const totalSessionsAttendedRead = await sessionAttendeesQueries.countEnrolledSessions(id, tenantCode)
```

Comment:
@sumanvpacewisdom if the user session Attended or sessions_hosted count changed we need to clear that user from cache